coconut/cmd/repo_default_revision.go

```diff
@@ -33,7 +34,14 @@ import (
 var repoDefaultRevisionCmd = &cobra.Command{
 	Use:   "default-revision",
 	Short: "set default global and per-repository revision",
-	Long: "The repository default-revision command sets the global default repository revision as well as the per-repository default revision.",
+	Long: fmt.Sprintf(`The repository default-revision command sets the global default repository revision.
```

Comment:
No need for `fmt.Sprintf` here

Reply:
That's true, as you're using `+` to concatenate rather than a format string. No big deal either way.

Reply:
Indeed, I thought that it was necessary for the line changes. I'll push a quick fix.